mscore/uploadscoredialog.cpp

```diff
@@ -78,6 +79,7 @@ UploadScoreDialog::UploadScoreDialog(LoginManager* loginManager)
 
       tagsHelp->setText(tr("Use a comma to separate the tags"));
```

Comment:
No, there is a notr=true in the UI file. And so all the help messages are in C++